features/util.py

```diff
@@ -100,8 +102,54 @@ def wait_for_boot(container_name: str) -> None:
         except ValueError:
             print("Unexpected runlevel output: ", process.stdout.strip())
             runlevel = None
-        if runlevel == "2":
+        if runlevel in ("2", "5"):
             break
         time.sleep(sleep_time)
     else:
         raise Exception("System did not boot in {}s".format(sum(retries)))
+
+
+def lxc_get_series(name: str, image: bool = False):
+    """Check series name of either an image or a container.
+
+    :param name:
+        The name of the container or the image to check its series.
+    :param image:
+        If image==True will check image series
+        If image==False it will check container configuration to get series.
+
+    :return:
+        The series of the container or the image.
+       `None` if it could not detect it (
+           some images doesn't have this field into properties).
```

Comment:
minor typos
some images don't have this field in properties).

Reply:
ouch. Sorry. Thanks